packages/atomic/src/components/ipx/atomic-ipx-modal/atomic-ipx-modal.pcss

```diff
@@ -0,0 +1,50 @@
+@import '../../../global/global.pcss';
+
+atomic-focus-trap {
+  @apply contents;
+}
+
+:host(.open) {
+  [part='backdrop'] {
```

Comment:
Might be better to use the ::part() selector in this file. If ever another part is added e.g. `<p part="part 1 part2" ` this will stop working